src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts

```diff
@@ -33,11 +36,13 @@ export class CreatePhotoFormComponent implements OnInit {
     Validators.minLength(ValidationConstants.INPUT_LENGTH_3),
     Validators.maxLength(ValidationConstants.MAX_DESCRIPTION_LENGTH_2000)
   ]);
+  SectionItemsFormArray = new FormArray([]);
 
   constructor(private formBuilder: FormBuilder, private store: Store ) {
     this.PhotoFormGroup = this.formBuilder.group({
       image: new FormControl(''),
       description: this.descriptionFormGroup,
+      sectionItemsFormArray: this.SectionItemsFormArray,
```

Comment:
this control in teh form group shoul be named exactly as it is in the model. In this case providerSectionItems
also, I am not sure how this filed is named on the backend side. The general idea - we have in the model exactly the same naming as it is in the backend. In the form-group we have exact same naming as in the model so while patching the form with values in edit mode - it won't have any problems. Otherwise you have to manage the correct value ptaching in edit mode

Reply:
same situation for workshop